app/src/main/java/com/teampophory/pophory/feature/HomeActivity.kt

```diff
@@ -0,0 +1,50 @@
+package com.teampophory.pophory.feature
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
+import com.teampophory.pophory.R
+import com.teampophory.pophory.databinding.ActivityHomeBinding
+import com.teampophory.pophory.feature.my.MyFragment
+import com.teampophory.pophory.feature.myElbum.MyDrawerFragment
+
+class HomeActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityHomeBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityHomeBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        setupBottomNavigationBar()
+        initializeDefaultFragment()
+    }
+
+    private fun setupBottomNavigationBar() {
+        binding.homeBottomNav.setOnItemSelectedListener { item ->
+            val selectedFragment = when (item.itemId) {
+                R.id.menu_my_drawer -> MyDrawerFragment()
+                R.id.menu_my -> MyFragment()
+                else -> null
+            }
+            selectedFragment?.let { changeFragment(it) }
+            return@setOnItemSelectedListener selectedFragment != null
+        }
+    }
+
+    private fun initializeDefaultFragment() {
+        val currentFragment = supportFragmentManager.findFragmentById(R.id.home_fcv)
+        if (currentFragment == null) {
+            val homeFragment = MyDrawerFragment()
+            changeFragment(homeFragment)
+        }
+    }
```

Comment:
```suggestion
    private fun initializeDefaultFragment(savedInstanceState: Bundle?) {
        if (savedInstanceState == null) {
            val homeFragment = MyDrawerFragment()
            changeFragment(homeFragment)
        }
    }
```

이번 기회에 savedInstanceState 알아보는 것도 좋을 듯. 왜 이렇게 변경을 제안 했을까에 대해 고민하는 것도 좋을 것 같아요

Hint) 프래그먼트 왔다갔다 하면서 다크모드 전환 한번 해보실래유?

Reply:
savedInstanceState에 활용에 대해서 처음 알게 되었는데 감사합니다 ㅎㅎ